src/scores/probability/brier_impl.py

```diff
@@ -65,3 +72,160 @@ def brier_score(
+    the Brier score is calculated as:
+
+
+    .. math::
+        s_{i,y} = \\left(\\frac{i}{m} - y\\right)^2
+
+    If ``fair_correction`` is set to ``True`` and the number of ensemble members is 1, the
+    fair correction will not be applied to avoid division by zero. If you want to
+    set the minimum number of ensemble members required to calculate the Brier score,
+    we recommend preprocessing your data to remove data points with fewer than the
+    minimum number of ensemble members that you want.
+
+    Args:
+        fcst: Real valued ensemble forecasts in xarray. The proportion of ensemble members
+        above a threshold is calculated within this function.
+        obs: Real valued observations in xarray. These values are converted to binary
+            values in this function.
+        ensemble_member_dim: The dimension name of the ensemble member.
+        event_thresholds: The threshold(s) that define what is considered an event. If
+            multiple thresholds are provided, they should be monotonically increasing with no NaNs.
+        reduce_dims: Optionally specify which dimensions to reduce when
+            calculating the Brier score. All other dimensions will be preserved.
+        preserve_dims: Optionally specify which dimensions to preserve when
+            calculating the score. All other dimensions will be reduced. As a
+            special case, 'all' will allow all dimensions to be preserved. In
+            this case, the result will be in the same shape/dimensionality
+            as the forecast, and the errors will be the absolute error at each
+            point (i.e. single-value comparison against observed), and the
+            forecast and observed dimensions must match precisely.
+        weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude)
+            when calculating the mean score across specified dimensions via ``reduce_dims`` or ``preserve_dims``.
+        fair_correction: Whether or not to apply the fair Brier score correction. Default is True.
+        event_threshold_mode: The mode to use to define an event relative to a supplied threshold.
+            Default is ``operator.ge``, which means that an event occurs
+            if the observation is greater than or equal to the threshold.
+            Similarly, an event will have been considered to be forecast by an ensemble member
+            if the forecast is greater than or equal to the threshold.
+            Passing in ``operator.lt`` will give the same results. Alternatively, you can provide
+            ``operator.gt`` which means that an event occurs if the observation
+            is strictly greater than the threshold. Again, an event for an ensemble member
+            will have been considered to be forecast if the forecast is greater than the threshold.
+            Using``operator.le`` will give the same results as ``operator.gt``.
```

Comment:
Consider "passing in ``operator.le`` is mathematically equivalent to ``operator.gt`` for this function".

Reply:
ping @nicholasloveday I think this may have gotten lost

Reply:
Updated